fl_training.py

```diff
@@ -149,191 +194,169 @@ def compute_test_score(
     y_val_global,
     x_test,
     y_test,
+    aggregation_weighting="uniform",
+    minibatch_count=20,
```

Comment:
Yes we do need it here. This method is not always used with a scenario perspective, it can also be used independently to compute the distributed learning on any set of nodes.